example.py

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python
-
+# import sys
+# sys.settrace()
```

Comment:
I will perform a complete review / test later today (if I have time, else later this week), but you can already remove those commented imports :)

Reply:
Thanks! Removed.